Gemfile

```diff
@@ -12,11 +12,11 @@ gem 'hanami-validations', '1.1.0.beta1', require: false, git: 'https://github.co
 gem 'hanami-router',      '1.1.0.beta1', require: false, git: 'https://github.com/hanami/router.git',      branch: 'develop'
 gem 'hanami-controller',  '1.1.0.beta1', require: false, git: 'https://github.com/hanami/controller.git',  branch: 'develop'
 gem 'hanami-view',        '1.1.0.beta1', require: false, git: 'https://github.com/hanami/view.git',        branch: 'develop'
-gem 'hanami-model',       '1.1.0.beta1', require: false, git: 'https://github.com/hanami/model.git',       branch: 'develop'
+gem 'hanami-model',       require: false, git: 'https://github.com/hanami/model.git',       branch: 'develop'
```

Comment:
@jodosha can we merge this PR with this branches and update it after in develop branch?